src/test/java/com/st/utopia/agent/UtopiaAgentApplicationTests.java

```diff
@@ -0,0 +1,16 @@
+package com.st.utopia.agent;
```

Comment:
If you're planning to put your tests in this file, well and good; if not, remove it.

Reply:
Yes, however, at this moment. This file makes sure that my project can at least run. It is a good initial testing file.